docs/releases/5.0.txt

```diff
@@ -612,3 +612,10 @@ to remove usage of these features.
 * Passing ``nulls_first=False`` or ``nulls_last=False`` to ``Expression.asc()``
   and ``Expression.desc()`` methods, and the ``OrderBy`` expression is no
   longer allowed.
+
+Features removed without deprecation.
```

Comment:
There is no need for a separate header. Features removed without a deprecation should be described in the [Backwards incompatible changes in 5.0](https://docs.djangoproject.com/en/dev/releases/5.0/#backwards-incompatible-changes-in-5-0).